web/custom-plugins/forecast-of-response/src/components/Operations/PhaseView.tsx

```diff
@@ -123,15 +169,12 @@ function NewSimulationConfig(props: { defaultVars: any }) {
           margin: '10px',
         }}
       >
-        <Button
-          variant="outlined"
-          color="danger"
-          onClick={() => setVariables(defaultVars)}
-        >
+        <Button variant="outlined" color="danger" onClick={() => resetValues()}>
```

Comment:
This button does not work. Likely because the <Input> does not read it's value from 'variables'.

Reply:
Fixed!